### DISCUSSION TITLE
Where find out all command in visual mode?

### DISCUSSION THREAD

**Discussion by stealzy** [gdh1995/vimium-c#696] [Q&A] [answered]
Found `E`, `B`, `W`, `AS`, did I miss something?
Probably stupid question, but I'm not Vim user ¯\\_(ツ)_/¯

**Comment by gdh1995** [comment, marked as answer]
A brief introduction is on https://github.com/philc/vimium/wiki/Visual-Mode ; while all the keys are defined in code on:
https://github.com/gdh1995/vimium-c/blob/f89f1b853ca7a69a27f4b6b311b7ef8ee02923b9/background/key_mappings.ts#L789-L814
